### ISSUE TITLE
Web accessibility gap found in Clarity wizard example.

### ISSUE THREAD

**Issue by pavandixit93** [vmware/clarity#4835]
## Describe the accessibility issue

_If this is a VMware product, do not fill this out! Instead consult with the accessibility team about the issue so we can consolidate all VMware related issues and track them separately._

_Provide clear description of the accessibility issue and the impact to the user. Is this only in certain use cases? Is it confusing or broken?_

As per observation,
Clarity wizard component interally Uses, `nav` and `main` HTML tag. As per the [Wcag W3](https://www.w3.org/TR/wai-aria-practices-1.1/examples/landmarks/main.html) Each page should have one main landmark.
Also If a page includes more than one `nav` landmark, each should have a unique label(https://www.w3.org/TR/wai-aria-practices-1.1/examples/landmarks/navigation.html).

## WCAG Guideline

_What WCAG guideline is being violated here and how? You should reference a guideline from 

[Each page should have one main landmark.
](https://www.w3.org/TR/wai-aria-practices-1.1/examples/landmarks/main.html ):  Each page should have one main landmark.

[more than one `nav` landmark
](https://www.w3.org/TR/wai-aria-practices-1.1/examples/landmarks/navigation.html ):  If a page includes more than one `nav` landmark, each should have a unique label

## How to reproduce
Issue found and reprodicible on https://clarity.design/documentation/wizards#examples, Open any Wizard example and Check HTML structure.(Please have a look at Attached, clarity_wizard_accessibility_issue.png)
![clarity_wizard_accessibility_issue](https://user-images.githubusercontent.com/21095558/88913348-5a3ed180-d27e-11ea-8643-2b68641f852e.png)

_Please provide a link to a reproduction scenario using one of the [Clarity Stackblitz templates](https://stackblitz.com/@clr-team). Reports without a clear reproduction may not be prioritized until one is provided._

1.  Go to '...'
2.  Click on '....'
3.  Scroll down to '....'
4.  See error

## Expected behavior

_A clear and concise description of what you expected to happen._

## Versions

**App**

- Angular: [e.g. 6]
- Node: [e.g. 8.10.0]
- Clarity: [e.g. 0.12.5]

**Device:**

- Type: [e.g. MacBook]
- Screen Reader: [e.g. VoiceOver, NVDA]

For screen readers, we primarily support the following combinations, and other combinations will be considered on a case by case basis.

- NVDA + Firefox (Windows)
- VoiceOver + Safari (Apple)

## Additional notes

_Add any other notes about the problem here._
Thank you

**Comment by gnomeontherun** [comment]
Clarity cares deeply about accessibility, and we work with our accessibility team to identify and triage issues. We have an internal tracking system to identify and verify accessibility issues, which has captured this already. Some issues with Clarity Angular are difficult to solve without causing side effects or breaking changes, and we have to balance concerns with fixing issues and breaking changes. As we build out Clarity Core, we are able to address these root cause accessibility issues and recommend adopting the Clarity Core implementations as they become available. We are closing this issue as we have the issues tracked internally. Please follow our development and releases to see when we release relevant Clarity Core components to make this possible.
